README.md-tpl

```diff
@@ -82,59 +83,337 @@ Generate an API key by signing in to
 [GOV.UK Notify](https://www.notifications.service.gov.uk) and going to
 the **API integration** page.
 
-## Send a message
-
-Text message:
+## Send messages
+### Text message
 
 ```java
-NotificationResponse response = client.sendSms(templateId, mobileNumber);
+HashMap<String, String> personalisation = new HashMap<>();
+personalisation.put("name", "Jo");
+personalisation.put("reference_number", "13566");
+SendSmsResponse response = client.sendSms(templateId, mobileNumber, personalisation, "yourReferenceString");
 ```
 
-Email:
+<details>
+<summary>
+SendSmsResponse
+</summary>
+
+If the request is successful, the SendSmsResponse is returned from the client. Attributes of the SendSmsResponse are listed below.
 
 ```java
-NotificationResponse response = client.sendEmail(templateId, emailAddress);
+    UUID notificationId;
+    Optional<String> reference;
+    UUID templateId;
+    int templateVersion;
+    String templateUri;
+    String body;
+    Optional<String> fromNumber;
+
 ```
 
-Find `templateId` by clicking **API info** for the template you want to send.
+Otherwise the client will raise a `NotificationClientException`:
+
+<table>
+<thead>
+<tr>
+<th>message</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>
+<pre>
+Status code: 400 {
+"errors":
+[{
+    "error": "TooManyRequestsError",
+    "message": "Exceeded send limits (50) for today"
+}]
+}
```

Comment:
this (and other exception messages) should be some java code detailing how to access the attributes on the `NotificationclientException` rather than a json-esque blob